### ISSUE TITLE
ModuleNotFoundError: No module named 'conda'

### ISSUE THREAD

**Issue by XiaoPeng24** [conda/conda#9401]
### Steps to Reproduce

Today I have to install the pytorch under my Anaconda environment, when I installed the pytorch using the command below:

`conda install pytorch
`

one of the required updated package is the **python 3.7** (my **previous python version is 3.6.8**), I enter 'yes', and everything just went Ok until the installation finished.

But when I use the `conda` command like `conda list`, `conda -V`, it has an exception below:

> Traceback (most recent call last):
    &ensp;   File "/home/lixinyue/anaconda3/bin/conda", line 12, in <module>
    &emsp; from conda.cli import main
   ModuleNotFoundError: No module named 'conda

and all of my previous package like tensorflow etc. cannot be imported. 
I refered this issue #2463, and in [here](https://github.com/conda/conda/issues/2463#issuecomment-520235329)  it says it is because the 

> the python3.6 was replaced by python3.7 and the base environment was corrupted

the commenter gives his solution, but I am still not clear how to solve this problem. What I want is to recover my old anaconda environments, because it has so many libs like, tensorflow, keras etc. 

Does anybody can give me some solutions or suggestions?

## Expected Behavior

Some solutions to recover my anaconda environment settings.

## Environment Information
<details open><summary><code>`conda info`</code></summary><p>

```
Ubuntu 18.04
```
</p></details>

<details open><summary><code>`conda config --show-sources`</code></summary><p>

```
Cannot use the 'conda config' command
```
</p></details>

<details><summary><code>`conda list --show-channel-urls`</code></summary><p>

```
Cannot use the 'conda list' command
```
</p></details>

**Comment by XiaoPeng24** [comment]
Today I have to install the pytorch under my Anaconda environment, when I installed the pytorch using the command below:

`conda install pytorch
`

one of the required updated package is the **python 3.7** (my **previous python version is 3.6.8**), I enter 'yes', and everything just went Ok until the installation finished.

But when I use the `conda` command like `conda list`, `conda -V`, it has an exception below:

> Traceback (most recent call last):
    &ensp;   File "/home/lixinyue/anaconda3/bin/conda", line 12, in <module>
    &emsp; from conda.cli import main
   ModuleNotFoundError: No module named 'conda

and all of my previous package like tensorflow etc. cannot be imported. 
I refered this issue #2463, and in [here](https://github.com/conda/conda/issues/2463#issuecomment-520235329)  it says it is because the 

> the python3.6 was replaced by python3.7 and the base environment was corrupted

the commenter gives his solution, but I am still not clear how to solve this problem. What I want is to recover my old anaconda environments, because it has so many libs like, tensorflow, keras etc. 

Does anybody can give me some solutions or suggestions?
